src/Sarif.Viewer.VisualStudio.Interop/SarifViewerInterop.cs

```diff
@@ -79,6 +79,8 @@ public bool IsViewerExtensionInstalled
         {
             get
             {
+                ThreadHelper.ThrowIfNotOnUIThread();
```

Comment:
Curious: Did these analyzers just recently kick in?

Reply:
Yes. Even though we are using a 15-level SDK, I've discovered that it's implicitly pulling in 16-level analyzers.

---
In reply to: [509536067](https://github.com/microsoft/sarif-visualstudio-extension/pull/228#discussion_r509536067) [](ancestors = 509536067)